pkg/netdevice/netResourcePool.go

```diff
@@ -71,17 +71,33 @@ func (rp *netResourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.Devic
 // StoreDeviceInfoFile stores the Device Info files according to the
 //  k8snetworkplumbingwg/device-info-spec
 func (rp *netResourcePool) StoreDeviceInfoFile(resourceNamePrefix string) error {
+	var devInfo nettypes.DeviceInfo
 	for id, dev := range rp.GetDevicePool() {
 		netDev, ok := dev.(types.PciNetDevice)
 		if !ok {
 			return fmt.Errorf("storeDeviceInfoFile: Only pciNetDevices are supported")
 		}
-		devInfo := nettypes.DeviceInfo{
-			Type:    nettypes.DeviceInfoTypePCI,
-			Version: nettypes.DeviceInfoVersion,
-			Pci: &nettypes.PciDevice{
-				PciAddress: netDev.GetPciAddr(),
-			},
+
```

Comment:
nit: we could probably roll this logic into `PciNetDevice` but it can be regarded as future improvement if we see this logic keeps expanding.

Reply:
I'll create an issue so we don't forget about it.

Reply:
#405